RetailSuiteAccounting/Overview/02_Features.md

```diff
@@ -0,0 +1,47 @@
+# Key Features
+
+Fully integrated financial accounting with the Actindo *Accounting* module:
+
+- Accounts receivable and accounts payable  
+  The Actindo *Accounting* module offers all necessary features for your accounting, including an integrated payment system and the automatic reconciliation of open payments.
+
+[comment]: <> (Preparation of VAT weggelassen. Evtl. RS FH)
+
+- Double-entry accounting  
+  Our *Accounting* module offers a real double-entry accounting based on DATEV with debit/credit entry and dialog as well as a batch and a quick entry mask.
+
+- General ledger and subledger  
+  The Actindo *Accounting* module can be used as a general ledger or as a subledger to synchronize data into the general ledger of your corporation.
+
+- Document verification including payment proposal list  
+  The module has an extended process for managing incoming invoices with release logic, comment function and the 4-eyes principle check. Based on the payment conditions of your suppliers, you can also receive payment suggestions.
+
+- Standard and customizable accounts chart  
+  DATEV-compatible charts of accounts SKR03 and SKR04 with support for additional, individual charts of accounts. All charts of accounts are fully customizable in the Actindo *Accounting* module, including tax keys and distribution logic.
+
+- Extensive evaluations  
+  Various evaluations such as profit and loss accounts, business assessments, cost center and cost unit accounting as well as profit center or the monthly balance sheet can be individually configured and created in the module.
+
+- Accounting Exports  
+  Data can be exported in standardized format for DATEV, SAP FICO, GDPdU or configured individually. Mapping functions for export to third-party systems are also available.
+
+- Fully Automatic Dunning  
+  The system recognizes open receipts and sends out fully automatic reminders.
+
+
+## Modern accounting
+
+**Digital workflows in accounting**  
+Folders full of paper invoices are a thing of the past. However, just digitizing transmission and processing of invoices is not enough. Efficient digitization of accounting involves automating everyday workflows, such as payment reconciliation, document verification and reminders. Automated processes will not only save you time, but also avoid errors that are common when working manually.
+
+## Data export made easy
+
+**Flexibility through interfaces**  
+Before switching to a Digital Operations Platform, companies usually already use other IT systems and isolated applications, including accounting tools. To avoid having to completely replace these, you can connect service providers such as DATEV or SAP to the Actindo Core1 Platform and easily export your data. GDPdU-certified exports for tax officials are also possible with the help of our *Accounting* module.
```

Comment:
```suggestion
Before switching to a Digital Operations Platform, companies usually already use other IT systems and isolated applications, including accounting tools. To avoid having to completely replace these, you can connect service providers such as DATEV or SAP to the *Actindo Core1 Platform* and easily export your data. GDPdU-certified exports for tax officials are also possible with the help of our *Accounting* module.
```